NEWS

```diff
@@ -1,6 +1,11 @@
 shiny-server 1.5.0
 --------------------------------------------------------------------------------
-* Add auto-reconnect capabilities.
+* Add auto-reconnect capabilities. Can be disabled via `disable_reconnect`
```

Comment:
It's super awkward. It's written this way so that it will make sense when it's enabled by default (next release) which will be more important for the majority of the lifetime of this feature, so I'm tolerating some awkwardness now rather than having to deprecate a config setting in the coming release.